src/modules/url-preview/url-preview.js

```diff
@@ -25,13 +25,6 @@ async function addOEmbed(id, item, links) {
 				}
 			});
 
-			if (!preview.description) {
```

Comment:
@aravindet This should be done on client. Client doesn't show author name currently, but if it starts showing, you will have it repeated twice.

Though I find it very unlikely that someone is bothering to include the author name in oembed data and not even set a description.